github/Repository.py

```diff
@@ -1772,6 +1774,7 @@ def edit(
         assert description is github.GithubObject.NotSet or isinstance(description, str), description
         assert homepage is github.GithubObject.NotSet or isinstance(homepage, str), homepage
         assert private is github.GithubObject.NotSet or isinstance(private, bool), private
+        assert visibility is github.GithubObject.NotSet or isinstance(visibility, str), visibility
```

Comment:
Can we assert here `visibility` has only the allowed values?
```suggestion
        assert visibility is github.GithubObject.NotSet or visibility in [
            "public", "private"
        ], visibility
```